src/components/misc/importer/settings/PersonOrganizationColumnSettings.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { FormattedMessage as Msg } from 'react-intl';
+import { connect } from 'react-redux';
+
+import SelectInput from '../../../forms/inputs/SelectInput';
+import { retrievePersonTags } from '../../../../actions/personTag';
+import { createSelection } from '../../../../actions/selection';
+import { getListItemsByIds } from '../../../../utils/store';
+import { flattenOrgs } from '../../../../utils/import';
+
+
+@connect(state => ({ 
+    subOrgs: state.subOrgs.items,
+    activeOrg: state.user.activeMembership.organization,
+}))
+export default class PersonOrganizationColumnSettings extends React.Component {
+    static propTypes = {
+        config: React.PropTypes.object.isRequired,
+        onChangeConfig: React.PropTypes.func,
+    };
+
+    constructor(props) {
+        super(props)
+        this.orgOptions = flattenOrgs(props.activeOrg, props.subOrgs);
+    }
+
+    render() {
+        let config = this.props.config;
+        let mappings = config.mappings
+            .sort((m0, m1) => (m0.value && m1.value)?
+                m0.value.toString().localeCompare(m1.value.toString()) : 0);
+
+        return (
+            <div className="PersonOrganizationColumnSettings">
+                <ul className="PersonOrganizationColumnSettings-mappings">
+                { mappings.map(mapping => {
+                    let value = mapping.value;
+                    let labelMsg = value ?
+                        'panes.import.settings.personOrganization.valueLabel' :
+                        'panes.import.settings.personOrganization.emptyLabel';
+
+                    return (
+                        <li key={ value || 0 }
+                            className="PersonOrganizationColumnSettings-mapping">
+                            <Msg tagName="h3" id={ labelMsg }
+                                values={{ value }} />
+                            <SelectInput
+                                value={ mapping.org }
+                                name={ value }
+                                options={ this.orgOptions }
+                                optionLabelsAreMessages={ false }
+                                onValueChange={ this.onOrganizationChange.bind(this) }
```

Comment:
There needs to be a null option here, so that we're not forced to assign all rows to an organization.

Alternatively, maybe we could rename the one that reflects the current organization so that it indicates that it's synonymous to picking null?

See screenshot below (which also shows the effect of the misspelled `emptyLabel` string mentioned elsewhere.)

![image](https://user-images.githubusercontent.com/550212/107369709-64ba0d80-6ae2-11eb-86f7-ac0fbfe2c294.png)

Reply:
Yeah that's pretty much what I intended, for the current organization to be the "null"-option.